python/example_code/lookoutvision/hello.py

```diff
@@ -0,0 +1,54 @@
+
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+# snippet-start:[python.example_code.lookoutvision.Hello]
+
+"""
+This example shows how to list your Amazon Lookout for Vision projects.
+If you haven't previously created a project in the current AWS Region,
+the response is an empty list, but does confirm that you can call the
```

Comment:
list, but does confirm --> list. However, it confirms

Reply:
Fixed.